dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -140,6 +149,11 @@ public abstract class Entry<S extends Entry, T extends Version> implements Compa
     @ApiModelProperty(value = "Implementation specific ID for the container in this web service", position = 0)
     private long id;
 
+    @JsonProperty("type")
+    @Transient
+    @Schema(requiredMode = RequiredMode.REQUIRED)
+    String type;
```

Comment:
A JavaDoc comment on what this used for would be helpful -- I already understand it in the context of this PR, but for future developers looking at this code.

Reply:
Also, it would be nice if we could combine `entryType` and `type`